joomla/joomla-cms
# [4.1] Allow save as copy article in the frontend

Pull Request for Issue # .

### Summary of Changes
This PR adds support for **Save As Copy** article from frontend of your site


### Testing Instructions
1. Apply Patch.
2. Login to frontend of your site using an account which has permission to edit an existing article
2. Edit the an existing article. Look at the Bottom of the page, if you are using an account which is having Create Article Permission, you will see a new button **Save As Copy** to allows you to create new article base on that existing article
3. Press Save As Copy button, confirm that the article is successfully created.
